catroid/src/main/java/org/catrobat/catroid/visualplacement/VisualPlacementActivity.java

```diff
@@ -124,6 +126,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
 	protected void onCreate(@Nullable Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.visual_placement_layout);
+		Bundle extras = getIntent().getExtras();
+		translateX = extras.getInt("X");
```

Comment:
"X" and "Y" should be constants in this class, and also used outside of here when creating the intent.